packages/create/create.js

```diff
@@ -624,7 +623,7 @@ async function inquirerMissingArgs(args) {
 			answers.publicAddress = args.publicAddress;
 		} else {
 			try {
-				let result = await phin("https://api.ipify.org/");
+				let result = await fetch("https://api.ipify.org/");
 				myIp = result.body.toString();
```

Comment:
This will cause `myIp` to become `"[object ReadableStream]"`. Need to use `await result.text()` here.

Reply:
Fixed in 47c5e0fabd3c0156d11dbe2400564df4fa2f890a